lib/faker/pizza.ex

```diff
@@ -0,0 +1,162 @@
+defmodule Faker.Pizza do
+  import Faker, only: [sampler: 2]
+
+  @moduledoc """
+  Functions for generating Pizza related data in English
+  """
+
+  @doc """
+  Returns a list with a random number of pizzas (in between the specified
+  range)
+  If no range is specified it defaults to 2..5
+  """
+  @spec pizzas(Range.t) :: list(String.t)
+  def pizzas(range \\ %Range{first: 2, last: 5})
+
+  def pizzas(%Range{first: first, last: last}) do
+    pizzas(:crypto.rand_uniform(first, last + 1))
+  end
+
+  @doc """
+  Returns a specified number of pizzas as a list
+  """
+  @spec pizzas(integer) :: list(String.t)
+  def pizzas(num) do
+    Stream.repeatedly(&pizza/0)
+    |> Enum.take(num)
+  end
+
+  @doc """
+  Returns a pizza
+  """
+  @spec pizza() :: String.t
+  def pizza, do: pizza(:crypto.rand_uniform(1, 31))
+  defp pizza(n) when n <= 5, do: "#{size_or_inches()} with #{toppings_and(n)}"
+  defp pizza(n) when n <= 10, do: "#{size_or_inches()} #{style()} with #{toppings_and(n - 5)}"
+  defp pizza(n) when n <= 15, do: "#{size_or_inches()} #{style()} #{combo()}"
+  defp pizza(_n), do: "#{size_or_inches()} #{combo()}"
+
+  @doc """
+  Returns a list with a random number of toppings (in between the specified
+  range)
+  If no range is specified it defaults to 2..5
+  """
+  @spec toppings(Range.t) :: list(String.t)
+  def toppings(range \\ %Range{first: 2, last: 5})
+
+  def toppings(%Range{first: first, last: last}) do
+    toppings(:crypto.rand_uniform(first, last + 1))
+  end
+
+  @doc """
+  Returns a specified number of toppings as a list
+  """
+  @spec toppings(integer) :: list(String.t)
+  def toppings(num) do
+    Stream.repeatedly(&topping/0)
+    |> Enum.take(num)
+  end
+
+  @doc """
+  Returns a string with a random number of toppings (in between the specified
+  range). "And" will appear before the last topping, along with an Oxford comma.
+  If no range is specified it defaults to 2..5
+  """
+  @spec toppings_and(Range.t) :: String.t
+  def toppings_and(range \\ %Range{first: 2, last: 5})
```

Comment:
`toppings_sentence` combined with renaming `add_and` to `to_sentence`. Inspired by rails' `to_sentence`